### PR TITLE
Use @timbl's vision description

### LINKED ISSUE #238: Add TBL vision bullets from https://team.inrupt.net/2020/Vision/TBL%20Solid%20Vision.html top home page at the top

### PR DESCRIPTION
**Pull request by Vinnl**
These bullets were written by Tim for the homepage.

Fixes #238.

### CODE REVIEW

File: pages/home.md [hunk 1]
```diff
@@ -48,13 +48,13 @@ permalink: /
...
-              <img src="{{site.baseurl}}/assets/img/like-peas-in-a-pod.svg" alt="[]" />
+              <img src="{{site.baseurl}}/assets/img/single-sign-on.svg" alt="[]" />
             </figure>
           </div>
           <div class="column is-offset-1 is-5">
-            <h2 class="title">Like peas in a Pod</h2>
+            <h2 class="title">The last login you'll ever need</h2>
             <p class="content is-large">
-              Your <a href="{{site.baseUrl}}/faqs#pod" title="Frequently Asked Questions - what is a Pod?">Pod</a> is where you store your data. Pods and apps are separate but compatible so you are free to change apps while keeping your data. For example, if you want to switch messaging app you can take your contacts and chat history along easily, as long as both apps support Solid.
+              Solid provides for the first time a <b>single global logon system</b>, so that when you log into any web site, instead of having  to log in with the usual 'f' and 'g' blue buttons, and then be tracked by Facebook and Google, you can log in with any Solid provider you trust, and that won't track you.
```
Comment (RubenVerborgh) [inline pages/home.md:57]: Not sure if we want to directly confront f and g.

File: pages/home.md [hunk 2]
```diff
@@ -64,14 +64,14 @@ permalink: /
       <div class="container">
         <div class="columns">
           <div class="column is-offset-1 is-5">
-            <h2 class="title">The last login you'll ever need</h2>
+            <h2 class="title">Like peas in a Pod</h2>
             <p class="content is-large">
-              Your <a href="{{site.baseUrl}}/faqs#webid" title="Frequently Asked Questions - what is a WebID?">WebID</a> changes the power dynamic. You call the shots: instead of creating an account for each app, have each app find your account. On top of that, your WebID makes it possible for people to connect to you, no matter what Solid apps they are using.
+              Solid provides separate places ("<a href="{{site.baseUrl}}/faqs#pod" title="Frequently Asked Questions - what is a Pod?">pods</a>") for you to store your data, a bit like Dropbox and GDrive and iCloud, but you have the <b>power to share</b> any file or folder or contact or event or whatever <b>with anyone who has a solid id</b>.  You store your data in any place you trust (including your own computer).
```
Comment (RubenVerborgh) [inline pages/home.md:69]:
```suggestion
              Solid provides separate places ("<a href="{{site.baseUrl}}/faqs#pod" title="Frequently Asked Questions - what is a Pod?">pods</a>") for you to store your data, a bit like Dropbox and Google Drive and iCloud, but you have the <strong>power to share</strong> any file or folder or contact or event or whatever <strong>with anyone who has a Solid ID</strong>.  You store your data in any place you trust (including your own computer).
```

File: pages/home.md [hunk 3]
```diff
@@ -87,18 +87,15 @@ permalink: /
           </div>
           <div class="column is-offset-1 is-5">
             <h2 class="title">
-              Open specifications built on open standards
+              Apps that give you the freedom to move
             </h2>
             <p class="content is-large">
+              When Solid apps start, they just store all the data they need to work in your pod.  They don't store the data themselves at some computer run by the person who owns the app.  So by default <b>you control all the data</b> in your solid world.  This is a massive improvement to the privacy world.
```
Comment (RubenVerborgh) [inline pages/home.md:93]:
```suggestion
              When Solid apps start, they just store all the data they need to work in your pod.  They don't store the data themselves at some computer run by the person who owns the app.  So by default <strong>you control all the data</strong> in your Solid world.  This is a massive improvement to your privacy.
```

File: pages/home.md [hunk 4]
```diff
@@ -87,18 +87,15 @@ permalink: /
...
+            </p>
+            <p class="content is-large">
+              Solid app builders don't just store the data in your pod any old way.  They use solid standard formats.  This means that you and the people you work with can actually use many different apps — at the same time even — to do stuff.
```
Comment (RubenVerborgh) [inline pages/home.md:96]:
```suggestion
              Solid app builders don't just store the data in your pod any old way.  They use Solid standard formats.  This means that you and the people you work with can actually use many different apps – at the same time even – to do stuff.
```

File: pages/home.md (hunk 1 above)
Comment (timbl) [reply pages/home.md]: Modified, to make them just examples

**Pull request merged**